hub/util/exceptions.py

```diff
@@ -731,3 +736,14 @@ def __init__(self, org_id, creds_key):
         super().__init__(
             f"Unable to find managed credentials '{creds_key}' for organization {org_id}."
         )
+
+
+class InvalidTokenException(Exception):
+    def __init__(self):
+        super().__init__("Token is invalid. Please check its spelling and try again.")
```

Comment:
`Token is invalid. Make sure the full token string is included and try again.` can be a better phrasing